apps/prairielearn/src/pages/studentAssessment/studentAssessment.html.ts

```diff
@@ -223,6 +223,7 @@ function GroupCreationJoinForm({
                     name="groupName"
                     maxlength="30"
                     placeholder="e.g. teamOne"
+                    aria-label="group name"
```

Comment:
It'd be better to change the `<h6>` above to a `<label>` and then use `for="..."` on it. That's not a heading anyways. Ditto for the join code below.

Reply:
Updated the `<h6>` here and below to `<label>` in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).